pandas/tests/indexes/datetimes/test_ops.py

```diff
@@ -12,32 +12,37 @@
     Index,
     PeriodIndex,
     Series,
+    TimedeltaIndex,
     Timestamp,
     bdate_range,
     date_range,
 )
 import pandas._testing as tm
-from pandas.tests.base.test_ops import Ops
 
 from pandas.tseries.offsets import BDay, BMonthEnd, CDay, Day, Hour
 
 START, END = datetime(2009, 1, 1), datetime(2010, 1, 1)
 
 
-class TestDatetimeIndexOps(Ops):
-    def setup_method(self, method):
-        super().setup_method(method)
-        mask = lambda x: (isinstance(x, DatetimeIndex) or isinstance(x, PeriodIndex))
-        self.is_valid_objs = [o for o in self.objs if mask(o)]
-        self.not_valid_objs = [o for o in self.objs if not mask(o)]
-
-    def test_ops_properties(self):
-        f = lambda x: isinstance(x, DatetimeIndex)
-        self.check_ops_properties(DatetimeIndex._field_ops, f)
-        self.check_ops_properties(DatetimeIndex._object_ops, f)
-        self.check_ops_properties(DatetimeIndex._bool_ops, f)
```

Comment:
Already tested in `pandas/tests/series/test_datetime_values.py::test_dt_namespace_accessor`

Reply:
Good catch.  test_dt_namespace_accessor could use a good refactor if youre up to it (separate PR)